app-privacy-policy.md

```diff
@@ -0,0 +1,180 @@
+
+
+## Browsing Online Content (ContentDB)
+
+You can find more information on
+[ContentDB's privacy policy](https://content.minetest.net/privacy_policy/).
+
+### Information collected
+
+Requests to [ContentDB](https://content.minetest.net) will be made when using
+the ContentDB dialog found at Content > Browse online content.
+The following information will be transferred or included:
+
+* IP address
+* Page URL
+* Minetest version
+* Platform, OS, device, etc
+
+ContentDB will know which packages you view and/or download.
+
+### How it is used
+
+HTTP requests are logged to aid with debugging, development, and aggregated
+statistics.
+
+Requests (such as downloads) are used for aggregated statistics and for
+calculating the popularity of packages. Download counts are shown for each
+package and release, and there are also download graphs available for each
+package.
+
+Whether an IP address has downloaded a package or release is cached to prevent
+downloads from being counted multiple times per IP address.
+
+### Who has access
+
+rubenwardy runs ContentDB and has access to the logs. Logs with personal
+information removed may be shared with others to aid with debugging or
+development.
+
+### Period of retention
+
+Logged HTTP requests are stored using log rotation, meaning that any requests
+will be forgotten within a few weeks.
+
+### Location
+
+ContentDB is currently located in Germany.
+By using this service, you give permission for the data to be moved as needed.
+
+### Removal requests
+
+See [ContentDB's privacy policy](https://content.minetest.net/privacy_policy/#removal-requests).
+
+
+## Version checking
+
+### Information collected
+
+When you open Minetest, it may contact www.minetest.net to fetch information
+about the most recent version. The following information will be transferred or
+included:
+
+* IP address
+* Minetest version
+* Platform, OS, device, etc
+
+### How it is used
+
+The version information is hosted on GitHub Pages.
+See [About GitHub Pages > Data collection](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#data-collection):
+
+> When a GitHub Pages site is visited, the visitor's IP address is logged and
+> stored for security purposes, regardless of whether the visitor has signed
+> into GitHub or not. For more information about GitHub's security practices,
+> see
+> [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).
+
+
+## Online play
+
+Minetest allows you to play online on multiplayer game servers. Please note that
+game servers are third-party, Minetest acts like a web browser connecting to a
+website. Therefore, please refer to the game server's privacy policy.
+
+When connecting to a game server, the following information is sent:
+
+* IP address
+* Minetest version
+* Window size, display, and input information
+* Anything you do in-game (including chat messages)
```

Comment:
could remove this section, it's likely to become out of date and we're shifting this responsibility anyway.

Reply:
As, we should remove the exact information and just mention to check the game server's privacy policy? I'm fine with that